optuna/samplers/_partial_fixed.py

```diff
@@ -0,0 +1,122 @@
+from typing import Any
+from typing import Dict
+import warnings
+
+import numpy
+
+from optuna.distributions import BaseDistribution
+from optuna.samplers import BaseSampler
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+
+
+class PartialFixedSampler(BaseSampler):
+    """Sampler that can sample with some parameters fixed.
+
+        .. versionadded:: 2.2.0
+
```

Comment:
It is standard in Optuna to use `experimental decorator` instead of `.. versionadded::`. How about the following?
```suggestion

@experimental("2.3.0")
class PartialFixedSampler(BaseSampler):
    """Sampler that can sample with some parameters fixed.

```

Reply:
You need to import `experimental` as follows.
```python
from optuna._experimental import experimental
```

Reply:
Sorry for the confusion. According to the [wiki](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#write--versionadded--directive-to-the-doc-of-new-features), we should add `.. versionaddded::`. Could you fix as follows?